src/model_repository_manager.cc

```diff
@@ -2156,20 +2153,17 @@ ModelRepositoryManager::DependencyGraph::Writeback(
     const std::set<ModelIdentifier>& affected_models)
 {
   for (const auto& model_id : affected_models) {
-    // An affected model can be deleted, so the presence of the model must be
-    // checked.
-    auto* updated_node = updated_dependency_graph.FindNode(
-        model_id, false /* allow_fuzzy_matching */);
-    if (updated_node != nullptr) {
-      auto* node = FindNode(model_id, false /* allow_fuzzy_matching */);
-      // Writeback
-      node->status_ = updated_node->status_;
-      node->checked_ = updated_node->checked_;
-      node->loaded_versions_ = updated_node->loaded_versions_;
-      node->is_locked_ = updated_node->is_locked_;
-      // Notify retry(s)
-      node->retry_notify_cv_->notify_all();
-    }
+    auto* node = GetNode(model_id);
+    auto* updated_node = updated_dependency_graph.GetNode(model_id);
+    // Writeback
+    node->status_ = updated_node->status_;
+    node->checked_ = updated_node->checked_;
+    node->loaded_versions_ = updated_node->loaded_versions_;
+    node->is_locked_ = updated_node->is_locked_;
```

Comment:
Would a simple copy constructor override make sense here?

Reply:
Maybe a `Writeback()` for nodes, so the `// Writeback` and `// Notify retry(s)` parts can be grouped?

Reply:
Sure, whatever you think makes most sense. I think writeback for nodes aligns with current writeback for graph and modelinfo

Reply:
Updated with a `Writeback()` for nodes.